src/network.zig

```diff
@@ -29,17 +29,17 @@ const Socket = struct {
 		switch(err) {
 			ws2.WSASYSNOTREADY => return error.WSASYSNOTREADY,
 			ws2.WSAVERNOTSUPPORTED => return error.WSAVERNOTSUPPORTED,
-			ws2.WSAEINPROGRESS => return error.WSAEINPROGRESS,
+			ws2.WSAEINPROGRESS => return error.BlockingOperationInProgress,
 			ws2.WSAEPROCLIM => return error.WSAEPROCLIM,
 			ws2.WSAEFAULT => return error.WSAEFAULT,
```

Comment:
This can also be unreachable, since it only indicates that we passed a bad address which is a bug.